transports/janus_rabbitmq.c

```diff
@@ -973,7 +1035,19 @@ void *janus_rmq_out_thread(void *data) {
 		return NULL;
 	}
 	JANUS_LOG(LOG_VERB, "Joining RabbitMQ out thread\n");
+	guint rmq_reconnect_backoff = rmq_reconnect_backoff_initial;
 	while(!rmq_client->destroy && !g_atomic_int_get(&stopping)) {
+
+		while (!rmq_client->connected) {
```

Comment:
the thread will be blocked forever in the loop when trying to destroy the transport